data/tlg4013/tlg001/tlg4013.tlg001.opp-grc1.xml

```diff
@@ -51,33 +51,36 @@
       </titleStmt>
       <publicationStmt>
 <authority>University of Leipzig</authority>
-<idno type="filename">TO BE ADDED BY LEIPZIG</idno>
+<idno type="filename">tlg4013.tlg001.opp-grc1.xml</idno>
 <availability>
  <p>Available under a Creative Commons Attribution-ShareAlike 4.0 International
  License</p>
 </availability>
-<date>2014</date>
+<date>2016</date>
 <publisher>University of Leipzig</publisher>
 <pubPlace>Germany</pubPlace>
 </publicationStmt>
 <sourceDesc>
-<listBibl xml:lang="lat">
+<listBibl>
  <biblStruct>
  <monogr>
- <title>COMMENTARIA IN ARISTOTELEM GRAECA</title>
+   <title xml:lang="lat">Simplicii in Aristotelis De Caelo Commentaria</title>
  <editor>
 	<persName>
-	 <name xml:lang="lat">I.L. HEIBERG</name>
+	 <name xml:lang="lat">I.L. Heiberg</name>
```

Comment:
I don't think the `<name>` element here needs an `xml:lang="lat"` attribute. Since this is the same as it would look in English, I think the `xml:lang` attribute can simply be deleted.